kale/embed/decomposition.py

```diff
@@ -3,19 +3,19 @@
 #         Haiping Lu, h.lu@sheffield.ac.uk or hplu@ieee.org
 # =============================================================================
 
-"""Python implementation of Multilinear Principal Component Analysis (MPCA)
-
-Reference:
-    Haiping Lu, K.N. Plataniotis, and A.N. Venetsanopoulos, "MPCA: Multilinear Principal Component Analysis of Tensor
-    Objects", IEEE Transactions on Neural Networks, Vol. 19, No. 1, Page: 18-39, January 2008. For initial Matlab
-    implementation, please go to https://uk.mathworks.com/matlabcentral/fileexchange/26168.
+"""Python implementation of tensor decomposition approach Multilinear Principal Component Analysis (MPCA)
+    and matrix decomposition Maximum independence domain adaptation (MIDA）
```

Comment:
Consistency of capitalisation for acronyms (MIDA and MPCA differ).

MIDA is a matrix factorization approach, not a matrix factorization, right?